ras_party/controllers/account_controller.py

```diff
@@ -132,6 +133,38 @@ def post_respondent(party, tran, session):
     return respondent.to_respondent_dict()
 
 
+@with_db_session
+def change_respondent_enrolment_status(payload, session):
+    """
+    Change respondent enrolment status for business and survey
+    :param payload:
+    :param session:
+    :return:
+    """
+    change_flag = payload['change_flag']
+    business_id = payload['business_id']
+    survey_id = payload['survey_id']
+    respondent_id = payload['respondent_id']
+    logger.info("Attempting to change respondent enrolment",
+                respondent_id=respondent_id,
+                survey_id=survey_id,
+                business_id=business_id,
+                status=change_flag)
+    respondent = query_respondent_by_party_uuid(respondent_id, session)
+    if not respondent:
+        raise RasError("Respondent does not exist.", status=404)
+
+    enrolment = query_enrolment_by_survey_business_respondent(respondent_id=respondent.id,
+                                                              business_id=business_id,
+                                                              survey_id=survey_id,
+                                                              session=session)
+    enrolment.status = change_flag
+
+    category = 'DISABLE_RESPONDENT_ENROLMENT' if change_flag == 'DISABLED' else 'ENABLE_RESPONDENT_ENROLMENT'
+    for case in get_cases_for_collection_exercise(survey_id, business_id, respondent_id):
+        post_case_event(case['id'], business_id, category=category)
```

Comment:
The desc param is left as the default, do you want the event description to be 'Default description message'?